sync_ci/pkg/parser/regex_rules.go

```diff
@@ -0,0 +1,164 @@
+package parser
+
+import (
+	"encoding/json"
+	"errors"
+	"github.com/pingcap/log"
+	"io/ioutil"
+	"regexp"
+	"sync/atomic"
+	"time"
+	"unsafe"
+)
+
+var (
+	reRules  unsafe.Pointer
+	rulePath = "./regex_rules.json"
```

Comment:
pls update the makefile to support the install command.

Reply:
ok